src/CoreWCF.Http/tests/Services/TestBasicScenariosService.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Services
 {
-    [ServiceBehavior]
+    //[ServiceBehavior]
```

Comment:
you want this behavior to be there ?

Reply:
Oops, that was left over from before I had explicit tests. I'll fix that.